hack/verify-package-specs.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+DIR="$( cd "${REPO_ROOT}" && pwd )"
+WORK_DIR=$(mktemp -d -p "${DIR}")
+
+if [[ -z "${WORK_DIR}" || ! -d "${WORK_DIR}" ]]; then
+  echo "Could not create temp dir"
+  exit 1
+fi
+
+echo "Setting up environment..."
+if [[ -z "${RELEASE_TOOL_BIN:-}" ]]; then
+  if [[ -n "${GOBIN:-}" ]]; then
+    export RELEASE_TOOL_BIN="${GOBIN}"
+  else
+    GOPATH=$(go env GOPATH)
+    export RELEASE_TOOL_BIN="${GOPATH}/bin"
+  fi
+fi
+export PATH="${PATH}:${RELEASE_TOOL_BIN}"
+
+function cleanup {
+  rm -rf "${WORK_DIR}"
+  echo "Deleted temp working directory ${WORK_DIR}"
+}
+
+trap cleanup EXIT
+
+cd "${WORK_DIR}"
+
+dir_path="${REPO_ROOT}/cmd/krel/templates/latest"
+if [[ -d  ${dir_path} ]]; then
+  dir_names=$(find "${dir_path}" -mindepth 1 -maxdepth 1 -type d -exec basename {} \;)
+
+  echo "Directory names in ${dir_path}:"
+  echo "${dir_names}"
+
+  for dir_name in ${dir_names}; do
+    echo "Processing Package: ${dir_name}"
+    krel obs specs --spec-only --package="${dir_name}" --template-dir="${dir_path}"
```

Comment:
`krel` cannot auto-determine the version for `cri-tools`, at least at the moment. When building specs for the `cri-tools` package`, you have to manually specify the version using the `--version` flag.

Reply:
@cpanato Let me take a look today if we can somehow automatically determine the version.